tools/StandardAnchorTags/Program.cs

```diff
@@ -61,7 +61,7 @@ static async Task<int> Main(string owner, string repo, bool dryrun =false)
         {
             if ((token is not null) && (headSha is not null))
             {
-                await logger.BuildCheckRunResult(token, owner, repo, headSha);
+                var annotations = await logger.BuildCheckRunResult(token, owner, repo, headSha);
```

Comment:
Revert this change, as it appears to be unused? (Or emit the annotations somewhere?)